packages/opera.rb

```diff
@@ -28,5 +28,7 @@ def self.install
     FileUtils.mv 'usr/bin', "#{CREW_DEST_PREFIX}"
     FileUtils.mv 'usr/lib', "#{CREW_DEST_PREFIX}"
     FileUtils.mv 'usr/share', "#{CREW_DEST_PREFIX}"
+    system "echo 'if [[ $(ls /usr/bin/ | grep x-www-browser) = x-www-browser ]]; then echo x-www-browser is already set; else sudo ln -s #{CREW_PREFIX}/bin/opera /usr/bin/x-www-browser; fi' > script.la.tmp"
+    system "bash script.la.tmp"
```

Comment:
Why not just include the symlink in the package like below:
```
FileUtils.ln_s "#{CREW_PREFIX}/bin/opera", "#{CREW_DEST_PREFIX}/bin/x-www-browser"
```
In any event, `ln -s #{CREW_PREFIX}/bin/opera /usr/bin/x-www-browser` will fail since that directory is read-only.

Reply:
You're right, but some apps may use the absolute path to run it like `openbox`, how can we redirect it to /usr/local/bin/ ?

Reply:
Your `PATH` environment variable should already look for `/usr/local/bin` first so it should be fine.

Reply:
But openbox is run the command like `/usr/bin/x-www-browser: command not found`
If can't solve I think it's ok :)

Reply:
Can't we fix the path since we compile openbox from source?